docs/docs/index.md

```diff
@@ -98,6 +99,29 @@ Finally, Chutney is production ready and used in companies by two main types of
 * Cross-Application Testing Teams  
   Dedicated testing teams use Chutney across multiple interconnected applications. These teams focus on validating complex integrations and ensuring that end-to-end scenarios work seamlessly in large systems with various dependencies.
 
+# Docker
+
+Along each Chutney release, some built docker images are pushed to the [GitHub Chutney repo](https://github.com/orgs/Enedis-OSS/packages?repo_name=chutney){:target="_blank"} :
```

Comment:
404 in all the link below (remove "orgs")

Reply:
Checked on legacy repo, it's the right url but we haven't any packages for the moment.